src/index.js

```diff
@@ -0,0 +1,146 @@
+
+//select HTML Elements 
+
+const body = document.querySelector('body');
+const searchBtn = document.getElementById('weather');
+const farenheit = document.getElementById('farenheit');
+const city = document.getElementById('city');
+const increaseBtn = document.getElementById('increase');
+const decreaseBtn = document.getElementById('decrease');
+const emojis = document.getElementById('emojis');
+const search = document.getElementById('search');
+const sky = document.getElementById('sky');
+const skyEmoji = document.querySelector('select');
+const reset = document.getElementById('reset');
+const currentWeather = document.getElementById('current-weather');
+const app = document.getElementById('weather-app');
+
+//dictionary for weather emojis
+const weatherImages = {
```

Comment:
Just for housekeeping! I would suggest keeping variables like `weatherImages` in a seperate file and importing it so the `index.js` file can look a bit more concise and cleaner!